src/MongoDB/Query.c

```diff
@@ -183,49 +363,44 @@ HashTable *php_phongo_query_get_debug_info(zval *object, int *is_temp TSRMLS_DC)
 	*is_temp = 1;
 	intern = Z_QUERY_OBJ_P(object);
 
-	array_init_size(&retval, 6);
+	array_init_size(&retval, 3);
 
-	/* Avoid using PHONGO_TYPEMAP_NATIVE_ARRAY for decoding query and selector
+	/* Avoid using PHONGO_TYPEMAP_NATIVE_ARRAY for decoding filter and opts
```

Comment:
This contrasts with other debug functions, which typically have:

```
/* Use native arrays for debugging output */
state.map.root_type = PHONGO_TYPEMAP_NATIVE_ARRAY;
state.map.document_type = PHONGO_TYPEMAP_NATIVE_ARRAY;
```

This debug handler uses `phongo_bson_to_zval()`, which defaults to the `PHONGO_TYPEMAP_NONE` via `PHONGO_BSON_STATE_INITIALIZER`.